test/modules/contents.js

```diff
@@ -27,4 +28,11 @@ describe('Contents', () => {
     });
     contents.run(mockResults, done);
   });
+
+  it('should not explode when there are more than 115 files', done => {
```

Comment:
This test does actually fail if you remove the fix!  So I'm confident this sorted it.